colossus/src/main/scala/colossus/core/ServerDSL.scala

```diff
@@ -37,30 +39,173 @@ class DSLDelegator(server : ServerRef, _worker : WorkerRef, initializer: Initial
 //this is mixed in by Server
 trait ServerDSL {
 
-  def start(name: String, settings: ServerSettings)(initializer: WorkerRef => Initializer)(implicit io: IOSystem) : ServerRef = {
+  val ConfigRoot = "colossus.server"
+
+  /**
+    * Create a new Server, using only the defaults provided by the corresponding "colossus.server" config path.
+    * A Config object will be created via {{{ConfigFactory.load()}}}
+    *
+    * @param initializer
+    * @param io
+    * @return
+    */
+  def start()(initializer: WorkerRef => Initializer)(implicit io: IOSystem) : ServerRef = {
+    start(ConfigRoot)(initializer)
+  }
+
+  /**
+    * Create a new Server using the supplied configPath.  This configPath will be overlaid on top of the default "colossus.server"
+    * config path.
+    * A Config object will be created via {{{ConfigFactory.load()}}}
+    *
+    * @param configPath The path to the configuration.
+    * @param initializer
+    * @param io
+    * @return
+    */
+  def start(configPath : String)(initializer: WorkerRef => Initializer)(implicit io: IOSystem) : ServerRef = {
+    start(configPath, ConfigFactory.load())(initializer)
+  }
+
+  /**
+    * Create a new Server by loading its config from the specified configPath.
+    * This configPath will be overlaid on top of the default "colossus.server" config path.
+    *
+    * @param configPath The path to the configuration.
+    * @param config The Config source to query
+    * @param initializer
+    * @param io
+    * @return
+    */
+  def start(configPath : String, config : Config)(initializer: WorkerRef => Initializer)(implicit io: IOSystem) : ServerRef = {
+    val userPathObject = config.getObject(configPath)
+    val serverConfig = userPathObject.withFallback(config.getObject(ConfigRoot)).toConfig
+    val name = serverConfig.getString("name")
+    start(name, ServerSettings.extract(serverConfig), config)(initializer)
+  }
+
+  /**
+    * Convenience function for starting a server using the specified name and port.  This will be overlaid on top of the default
+    * "colossus.server" configuration path.
+    *
+    * @param name Name of this Server. Name is also the MetricAddress relative to the containing IOSystem's MetricNamespace
+    * @param port Port to run on
+    * @param initializer
+    * @param io
+    * @return
+    */
+  def start(name: String, port: Int)(initializer: WorkerRef => Initializer)(implicit io: IOSystem): ServerRef = {
+
```

Comment:
Feels a bit sneaky, but also allowed me to continue to have these functions all chain into each other